### PR TITLE
add Romanian STS dataset

### PR DESCRIPTION
**Pull request by rbroc**
## Checklist for adding MMTEB dataset

**Reason for dataset addition**: Adding Romanian STS dataset (machine-translated + human rectified version of English STSBenchmark).
Should be the first Romanian dataset overall.

**Checklist**:
- [x] I have tested that the dataset runs with the `mteb` package.
- [x] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb run -m {model_name} -t {task_name}` command.
  - [x] `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2`
  - [x] `intfloat/multilingual-e5-small`
- [x] I have checked that the performance is neither trivial (both models gain close to perfect scores) nor random (both models gain close to random scores).
- [x] I have considered the size of the dataset and reduced it if it is too big (2048 examples is typically large enough for most tasks)
- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`. 
- [x] I have added points for my submission to the [POINTS.md](https://github.com/embeddings-benchmark/mteb/blob/main/docs/mmteb/points.md) file.

### CODE REVIEW

File: mteb/tasks/STS/ron/RonSTS.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            "revision": "41a33183b739070f3d46d9d446492c1d2f98ce1a",
+        },
+        description="High-quality Romanian translation of STSBenchmark.",
+        reference="https://openreview.net/forum?id=JH61CD7afTv",
+        type="STS",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["ron-Latn"],
+        main_score="cosine_spearman",
+        date=None,
```
Comment (isaac-chung) [inline mteb/tasks/STS/ron/RonSTS.py:22]: Could use a rough estimate of the dates?

File: mteb/tasks/STS/ron/RonSTS.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        form=["written"],
+        domains=["News", "Social", "Web"],  # web for image captions
+        task_subtypes=None,
```
Comment (isaac-chung) [inline mteb/tasks/STS/ron/RonSTS.py:25]: Does this match anything listed in `mteb/abstasks/TaskMetadata.py`?

**Review by isaac-chung — APPROVED** [review]
Thanks for the contribution! Looks good. Just a few small comments. 
And if you could please +2 pts for review for me.

File: mteb/tasks/STS/ron/RonSTS.py (hunk 1 above)
Comment (rbroc) [reply mteb/tasks/STS/ron/RonSTS.py]: added a rough estimate for dataset creation now

File: mteb/tasks/STS/ron/RonSTS.py (hunk 2 above)
Comment (rbroc) [reply mteb/tasks/STS/ron/RonSTS.py:25]: nope. it's a semantic similarity task (no other STS task has values for this field)

**Comment by rbroc** [comment]
points added -- isn't PR review 1 point though? :)

File: mteb/tasks/STS/ron/RonSTS.py (hunk 1 above)
Comment (rbroc) [reply mteb/tasks/STS/ron/RonSTS.py]: there's really little info in the paper though, so it's a very rough estimate :)

**Comment by imenelydiaker** [comment]
@rbroc linting is not passing, maybe if you run `make lint` or `make pr` locally and push modified files it should be okay.

**Comment by rbroc** [comment]
@imenelydiaker @isaac-chung linting and all tests passing now, should be ready to merge

**Comment by rbroc** [comment]
@imenelydiaker @isaac-chung linting and all tests passing now, should be ready to merge

**Comment by isaac-chung** [comment]
@rbroc thanks again for your contribution 🤗

**Pull request merged**